src/foundry/client/applications/ux/drag-drop.d.mts

```diff
@@ -12,10 +14,141 @@
  * });
  * dragDrop.bind(html);
  * ```
- * @remarks TODO: Stub, copy from v12 implementation & update
  */
-declare class DragDrop {}
+declare class DragDrop {
+  /**
+   * @param options - (default: `{}`)
+   */
+  constructor({ dragSelector, dropSelector, permissions, callbacks }?: DragDrop.Configuration);
+
+  /**
+   * A set of callback functions for each action of the drag & drop workflow.
+   * @defaultValue `{}`
+   */
+  callbacks: InexactPartial<Record<DragDrop.Action, (event: DragEvent) => void>>;
+
+  /**
+   * The HTML selector which identifies draggable elements.
+   * @defaultValue `undefined`
+   */
+  dragSelector: string | null | undefined;
+
+  /**
+   * The HTML selector which identifies drop targets.
+   * @defaultValue `undefined`
+   */
+  dropSelector: string | null | undefined;
```

Comment:
I don't think the `undefined` here is accurate. The constructor is:
```ts
  constructor({dragSelector=null, dropSelector=null, permissions={}, callbacks={}}={}) {
    this.dragSelector = dragSelector;
    this.dropSelector = dropSelector;
    this.permissions = permissions;
    this.callbacks = callbacks;
  }
```

Therefore it's `string | null`, no `undefined`.